docs/pages/versions/unversioned/sdk/firebase-analytics.md

```diff
@@ -26,13 +26,13 @@ When using the web-platform, you'll also need to run `expo install firebase`, wh
 To use this package, the native Firebase configurations need to be added to your app.
 [Please follow this guide on how to set up native Firebase.](/guides/setup-native-firebase)
 
-## Expo client: Limitations & configuration
+## Expo Go: Limitations & configuration
 
-The use of Native Firebase Analytics requires that the google-services configuration is bundled and linked into your app. Since the standard Expo client loads projects on demand, it does not have the google-services configuration linked into its app-bundle.
+The use of Native Firebase Analytics requires that the google-services configuration is bundled and linked into your app. Since the standard Expo Go loads projects on demand, it does not have the google-services configuration linked into its app-bundle.
```

Comment:
```suggestion
The use of Native Firebase Analytics requires that the google-services configuration is bundled and linked into your app. Since Expo Go loads projects on demand, it does not have the google-services configuration linked into its app-bundle.
```